official/docs/csharp/events/RetrieveEventList.cs

```diff
@@ -0,0 +1,9 @@
+using EasyPost;
+
+EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");
+
+listParams = new Dictionary<string, object>() { "page_size", 5 };
```

Comment:
```suggestion
Dictionary<string, object> listParams = new Dictionary<string, object>() { "page_size", 5 };
```
You need to the type before the variable name `listParams`